src/dmd/backend/elpicpie.d

```diff
@@ -253,7 +255,7 @@ elem * el_var(Symbol *s)
         type_debug(t);
         e.ET = t;
         t.Tcount++;
-static if (TARGET_WINDOS)
+if (config.exe & EX_windos)
```

Comment:
indentation

Reply:
I know. I didn't change the indentation because it would make things much harder to review. A whitespace-only PR would be apropos later.